ak_vendor/templates/report_template.html

```diff
@@ -1317,6 +1321,27 @@ <h3 class="analysis__sectionheader">{% trans "Regulatory" %}</h3>
                 {% endif %}
               </div>
 
+              <div class="regulatory__block">
```

Comment:
so are we showing only pcidss3 for old analysis and only pcidss4 for new ones?

Reply:
yes, the backend will send both responses but the newer files will only be having the PCI-DSS 4.0.